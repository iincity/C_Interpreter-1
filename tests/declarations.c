#include <stdio.h>


int a=0, b=0, c=10;

int main()
{
	printf("%d\n", a);
	printf("%d\n", b);
	printf("%d\n", c);

	int d = 9, e = 10 + (b += 1), f = d *= 2;
	printf("%d\n", d);
	printf("%d\n", e);
	printf("%d\n", f);

	int a = 50;
	printf("%d\n", a);
	printf("%d\n", b);
	int b = 100;
	printf("%d\n", b);

	return 0;
}


