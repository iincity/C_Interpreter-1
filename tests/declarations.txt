
int a=0, b=0, c=10;

int main()
{
	print a;
	print b;
	print c;

	int d = 9, e = 10 + (b += 1), f = d *= 2;
	print d;
	print e;
	print f;

	int a = 50;
	print a;
	print b;
	int b = 100;
	print b;

	return 0;
}


