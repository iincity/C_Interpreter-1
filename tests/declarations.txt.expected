0
0
10
18
11
18
50
1
100
